docs/topics/multiplatform/multiplatform-hierarchy.md

```diff
@@ -116,114 +131,106 @@ to your gradle.properties
 Learn more about hierarchy templates: https://kotl.in/hierarchy-template
 ```
 
-The following few sections will discuss several popular cases where you might see this warning, and what you can do
-
-#### Replacing your manual configuration with the Default Hierarchy Template
-
-**Case.** All added dependsOn-edges are covered by the Default Hierarchy Template. This can happen, for example, if 
-you're migrating previously existing project to 1.9.20 or if you have used some code snippets written before 1.9.20.
+To solve this issue, configure your project by doing one of the following:
 
-**Solution.** Remove all the manual dependsOn-edges (`dependsOn`-calls) and instantiation of the intermediate source sets
-(`by creating`-calls). 
+* [Replace your manual configuration with the default hierarchy template](#replacing-manual-configuration)
+* [Add your source sets to the default hierarchy template](#adding-source-sets)
```

Comment:
Please don't use this wording. "Adding source sets to the default hierarchy template" 100% reads as a completely different action that actually *modified* the default hierarchy template. That is a frequent request, but it's explicitly forbidden for now. 

What we do here instead is that we create additional source sets *alongside* the ones created by the template. 

Please reformulate

Reply:
Changed to "Creating additional source sets"